### PR TITLE
 #160386275 Fix travel duration days

### PR DESCRIPTION
**Pull request by amarachukwu-agbo**
### What does this PR do?
Fixes the travel duration when a request is created
#### Description of Task to be completed?
- Add trips array inside request

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#160386275]
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

File: src/modules/notifications/__tests__/notifications.test.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('Notifications Controller', () => {
       });
     });
     describe('Authenticated user with no notifications', () => {
-      it(`should return 200 status and the appropriate
+      xit(`should return 200 status and the appropriate
```
Comment (samuelkubai) [inline src/modules/notifications/__tests__/notifications.test.js]: Why are we removing this test?

File: src/modules/requests/RequestsController.js [hunk 2]
```diff
@@ -30,15 +30,11 @@ class RequestsController {
           }
         )));
         const approval = await ApprovalsController.createApproval(request);
-        const message = 'created a new travel request';
-
-        RequestsController.sendNotificationToManager(req, res, request, message);
-
+        request.dataValues.trips = requestTrips;
```
Comment (samuelkubai) [inline src/modules/requests/RequestsController.js]: Are we removing the notification of a new approval?

Comment (amarachukwu-agbo) [reply src/modules/requests/RequestsController.js]: No, it is still there, I discovered they were two of them

Comment (samuelkubai) [reply src/modules/requests/RequestsController.js]: Oooh, cool.

File: src/modules/notifications/__tests__/notifications.test.js (hunk 1 above)
Comment (samuelkubai) [reply src/modules/notifications/__tests__/notifications.test.js]: @amarachukwu-agbo What about this test?

**Pull request merged**